### PR TITLE
Replaced gitter with matrix

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      |no
| New feature?  |no
| BC breaks?    |no
| Deprecations? |no
| Tests pass?   | yes
| Documentation |no
| Translation   | yes
| CHANGELOG.md  | no
| License       | MIT

### CODE REVIEW

File: translations/messages.pl.yml [hunk 1]
```diff
@@ -474,7 +474,7 @@ quickstart:
         description: Jeżeli potrzebujesz pomocy, jesteśmy tutaj dla ciebie.
         github: na GitHubie
         email: przez e-mail
-        gitter: na Gitterze
+        matrix: na Gitterze
```
Comment (Kdecherf) [inline translations/messages.pl.yml:477]: I'm not sure we should keep the translation in the languages were there is a doubt on the update to make. I would go with letting a missing translation entry in this situation

**Review by Kdecherf — CHANGES_REQUESTED** [review]

File: translations/messages.pl.yml (hunk 1 above)
Comment (nicosomb) [reply translations/messages.pl.yml:477]: you'd prefer to remove these lines?

Comment (Kdecherf) [reply translations/messages.pl.yml:477]: Yes I would prefer

Comment (nicosomb) [reply translations/messages.pl.yml]: Done.

File: translations/messages.cs.yml [hunk 2]
```diff
@@ -343,7 +343,7 @@ tag:
 quickstart:
     support:
         title: Podpora
-        gitter: Na Gitteru
+        matrix: Na Gitteru
```
Comment (Kdecherf) [inline translations/messages.cs.yml:346]: Remove

File: translations/messages.fa.yml [hunk 3]
```diff
@@ -270,7 +270,7 @@ quickstart:
         description: به کمک نیاز دارید؟ ما پشتیبان شما هستیم.
         github: روی گیت‌هاب
         email: با ایمیل
-        gitter: روی گیتر
+        matrix: روی گیتر
```
Comment (Kdecherf) [inline translations/messages.fa.yml:273]: Delete

**Review by Kdecherf — CHANGES_REQUESTED** [review]

**Pull request merged**